template.tpl

```diff
@@ -44,16 +44,16 @@ ___TEMPLATE_PARAMETERS___
       },
       {
         "args": [
-          "t2_[a-z0-9]+"
+          "(t2_|a2_)[a-z0-9]+"
```

Comment:
Were you able to test this in the tag manager?  I see the unit test -- which is super! -- but a test from the UI would be nice confirmation it works as intended.  If you haven't played with tagmanager.google.com before, I can show you.